newIDE/app/src/MainFrame/index.js

```diff
@@ -1365,6 +1365,13 @@ const MainFrame = (props: Props) => {
       // from the user.
       findAndLogProjectPreviewErrors(currentProject);
 
+      const fallbackAuthor = authenticatedUser.profile
+        ? {
+            username: authenticatedUser.profile.username || '',
+            id: authenticatedUser.profile.id,
+          }
+        : undefined;
```

Comment:
Ubernit: I tend to prefer `null` to convey the feeling of "this thing has a value, but it's null because it's not applicable" - while undefined can be a bug.

Reply:
True, I'll change it